rfq/order.go

```diff
@@ -95,9 +103,17 @@ type AssetSalePolicy struct {
 	// the policy.
 	MaxOutboundAssetAmount uint64
 
+	// CurrentAssetAmountMsat is the total amount that is held currently in
+	// accepted htlcs.
```

Comment:
nit: use the same capitalization for HTLC everywhere? So `s/htlcs/HTLCs` here and throughout the file.